packages/fxa-settings/src/components/UnitRowRecoveryKey/en-US.ftl

```diff
@@ -11,7 +11,7 @@ rk-cannot-remove-key = Your account recovery key could not be removed.
 rk-refresh-key = Refresh recovery key
 rk-content-explain = Restore your information when you forget your password.
 rk-content-reset-data = Why does resetting my password reset my data?
-rk-cannot-verify-session-3 = Sorry, there was a problem verifying your session
+rk-cannot-verify-session-4 = Sorry, there was a problem confirming your session
```

Comment:
I'm fine with leaving this for consistency but this could be renamed to `rk-cannot-confirm-session`, similar to a couple of others like `tfa-cannot-confirm-code` but it doesn't matter too much.

Reply:
I'm fine with that if you feel strongly about it! I just left it because there were so many var references to "verify" that I kind of gave up on updating any of them (since I can't do all of them)

Reply:
Yeah it's fine by me to just leave them.